configure.ac

```diff
@@ -57,10 +57,40 @@ AC_ARG_WITH([rootlibdir],
         [], [with_rootlibdir=$libdir])
 AC_SUBST([rootlibdir], [$with_rootlibdir])
 
+AC_ARG_WITH([systemdsystemunitdir],
+			AC_HELP_STRING([--with-systemdsystemunitdir=DIR],
+			[path to systemd system unit directory]),
+					[path_systemunitdir=${withval}])
+if (test "${enable_systemd}" != "no" && test -z "${path_systemunitdir}"); then
+	AC_MSG_CHECKING([systemd system unit dir])
+	path_systemunitdir="`$PKG_CONFIG --variable=systemdsystemunitdir systemd`"
```

Comment:
Nevertheless, this code **is not** working. It is trying to use `pkg-config` to get default values for those variables, but `PKG_CONFIG` is not set. You must use `PKG_PROG_PKG_CONFIG` to set it. See https://autotools.io/pkgconfig/pkg_check_modules.html